MUXSDKKaltura/MUXSDKKaltura/MUXSDKPlayerBinding.swift

```diff
@@ -395,16 +388,23 @@ public class MUXSDKPlayerBinding: NSObject {
         self.videoData.hasUpdates = false
     }
     
-    private func handleRenditionChange(bitrate: Double) {
-        guard self.videoData.lastAdvertisedBitrate != 0 else {
-            // Starting Playback
-            self.videoData.lastAdvertisedBitrate = bitrate
+    private func handleRenditionChange(event: AVPlayerItemAccessLogEvent) {
+        guard
+            self.videoData.lastAdvertisedBitrate != 0,
+            self.videoData.started
+        else {
+            self.videoData.hasUpdates = true
```

Comment:
I think because of the logic we have to register the first rendition played, we are going to need to move the `hasUpdates` logic for rendition to the `updateVideoData` method. Because if we set this to true but `videoData` gets called before playback has started we are going to ignore this change and in the end of the `updateVideoData` we would set hasUpdates back to false even though we never really registered the rendition change value so I think we should remove `self.videoData.hasUpdates = true` and from line 13 and just compare the `lastDispatchedAdvertisedBitrate` to `lastAdvertisedBitrate` within the `updateVideoDataForPlayer` to determine if there are rendition updates or not.